modules/ROOT/pages/deploy-index.adoc

```diff
@@ -8,10 +8,12 @@ Anypoint Runtime Fabric supports deployment to these versions of Mule runtime:
 
 * 3.8.7
 * 3.9.1, 3.9.2
-* 4.1.2+
```

Comment:
@sundippatel  @sarjeet2013 This list looks out of date/incorrect? Can we remove it and just have a link to the RNs?

Reply:
Thanks!